src/cpp/src/sequence_group.hpp

```diff
@@ -413,18 +420,29 @@ class SequenceGroup {
 
     void clear_scheduled_tokens() {
         m_num_scheduled_tokens = 0;
+        m_num_validated_tokens = 0;
     }
 
     bool is_scheduled() const {
         return m_num_scheduled_tokens > 0;
     }
 
+    void set_num_validated_tokens(size_t k) {
+        // in case of non-prompt we need to take prev tokens + token to validate
+        // m_num_validated_tokens = get_num_processed_tokens() ? k + 1 : k;
+        m_num_validated_tokens = k;
+    }
+
+    size_t get_num_validated_tokens() {
```

Comment:
validated => validation ? or get_num_tokens_to_validate ?
because validated means that they are confirmed by main models.. 

If you are OK, let's change validat**ed** in other places too

Reply:
applied